builder/dashboard.go

```diff
@@ -18,13 +18,15 @@ type Dashboard struct {
 type DashboardBuilder struct {
 	resourceNames  *ResourceNames
 	panelFactories []PanelFactory
+	environment    string
 	orchestrator   string
 }
 
-func NewDashboardBuilder(resourceNames *ResourceNames, orchestrator string) *DashboardBuilder {
+func NewDashboardBuilder(resourceNames *ResourceNames, environment string, orchestrator string) *DashboardBuilder {
```

Comment:
maybe add the environment to the resourceNames?

Reply:
That sounds a little misfitting to me, as it is none of our resources that we are handling but is more of a "fact" of which infrastructure we are running against.